app/views/layouts/application.html.erb

```diff
@@ -13,6 +13,38 @@
   </head>
 
   <body>
+    <nav class="navbar navbar-expand-lg navbar-light bg-light justify-content-between">
+      <a class="navbar-brand">My Own Blog</a>
+
+      <ul class="navbar-nav mr-auto">
+        <% if current_user.present? %>
+          <li class="nav-item">
+            <%= link_to(
+                  current_user.full_name.presence || current_user.email,
```

Comment:
As you have `to_s` method in `User` class with same code, you need to replace this.